pyproject.toml

```diff
@@ -16,6 +16,7 @@ pyyaml = "^5"
 dazl = "^7"
 daml-dit-api = "0.4.3"
 jwcrypto = "0.9.1"
+google-auth = "*"
```

Comment:
I originally made this an "optional" dependency because it's a somewhat expensive thing to pull in.

Probably worth looking into something like [this](https://stackoverflow.com/questions/6237946/optional-dependencies-in-distutils-pip) to see if there is a way we can avoid _everyone_ taking this dependency where it isn't needed.

There might even be a way for them to specify it on their dazl dependency…I'll take a look.

Reply:
`pip install dazl[oauth]` might do the trick:

https://github.com/digital-asset/dazl-client/blob/6ce8c1728999c414cee57bb909fa6e0631f84ff9/pyproject.toml#L53

So their project should list two dependencies (see [here](https://stackoverflow.com/questions/71386332/how-do-i-specify-extra-bracket-dependencies-in-a-pyproject-toml)):

```toml
daml_dit_if = "^0.6.7"
dazl = { extras = ["oauth"], version = "^7" }
```

Reply:
Yeah I think it's fair to require depending on this at the project level since using the token it's a less common use case, I'll change it.

Reply:
I'll have to see how the above you posted works with `requirements.txt`, since that is what `ddit` uses to install dependencies.

Reply:
Nice, just adding `dazl[oauth]` to `requirements.txt` in the integration project seems to work.